node/src/BaseClient.ts

```diff
@@ -1566,59 +1571,21 @@ export class BaseClient {
      *
      * const result2 = await client.bitpos("key1", 1, -1);
```

Comment:
if you use a separate @example tag for each example, you don't need result1, result2, etc
```
     * @example
     * ```typescript
```